packages/core/contracts/financial-templates/contract-for-difference/ContractForDifference.sol

```diff
@@ -198,8 +200,8 @@ contract ContractForDifference is Testable, Lockable {
             contractState = ContractState.ExpiredPriceReceived;
         }
 
-        require(longToken.burnFrom(msg.sender, longTokensToRedeem));
-        require(shortToken.burnFrom(msg.sender, shortTokensToRedeem));
+        require(longToken.burnFrom(msg.sender, longTokensToRedeem), "Redeeming long failed");
+        require(shortToken.burnFrom(msg.sender, shortTokensToRedeem), "Redeeming short failed");
```

Comment:
Similar point to the above. This will revert elsewhere, so is there a point to having the message (or even the require itself)?

Reply:
removed :)